modules/apps/depot/depot-web/src/main/java/com/liferay/depot/web/internal/display/context/DepotAdminMembershipsDisplayContext.java

```diff
@@ -72,6 +74,36 @@ public int getDepotGroupsCount() throws PortalException {
 		return groups.size();
 	}
 
+	public List<Group> getInheritedDepotGroups() throws PortalException {
+		if (_inheritedDepotGroups != null) {
+			return _inheritedDepotGroups;
+		}
+
+		_inheritedDepotGroups = Collections.emptyList();
+
+		if (_user == null) {
+			return _inheritedDepotGroups;
+		}
+
+		List<Organization> organizations = _user.getOrganizations();
```

Comment:
Hi @gergelyszalay ,

Is it possible to add also user group? 

is should be something similar to : 

```
List<UserGroup> userGroups = _user.getUserGroups();

if (userGroups.isEmpty()) {
	return _inheritedDepotGroups;
}

_inheritedDepotGroups.addAll(ListUtil.filter(
	GroupLocalServiceUtil.getUserGroupsRelatedGroups(userGroups),
	Group::isDepot));
``` 

and also you will have to take into account to do not show user group inherit asset libraries in inherit sites.

Reply:
thanks @Mikellorza , Now it is considering inherited AL from usergroups as well.